### PR TITLE
Swagger: support no type input. support non-struct type response

### PR DESCRIPTION
**Pull request by joejoe321321**

### CODE REVIEW

File: cmd/rdl-gen-parsec-swagger/main.go [hunk 1]
```diff
@@ -238,31 +239,35 @@ func swagger(schema *rdl.Schema, genParsecError bool, swaggerScheme string, fina
...
-		}
+
+		genResourceError(defs)
+
 		if genParsecError {
 			addParsecError(defs)
 		}
 		swag.Definitions = defs
-	}
+	//}
```
Comment (chinkong83) [inline cmd/rdl-gen-parsec-swagger/main.go]: 留一個 // } 有啥特別用意?

**Comment by chinkong83** [comment]
ok

File: cmd/rdl-gen-parsec-swagger/main.go (hunk 1 above)
Comment (joejoe321321) [reply cmd/rdl-gen-parsec-swagger/main.go]: 呃, 沒有, 應該是不小心按到的

**Comment by chinkong83** [comment]
ok

**Pull request merged**